Library/Homebrew/config.rb

```diff
@@ -27,7 +27,11 @@ def mkpath
 HOMEBREW_CACHE_FORMULA = HOMEBREW_CACHE+"Formula"
 
 unless defined? HOMEBREW_BREW_FILE
```

Comment:
Now that this question comes up again, I wonder why this is in `config.rb` at all. As far as I can tell, there are only two cases:

1. Homebrew is started normally via `bin/brew`. In this case `HOMEBREW_BREW_FILE` is set in `brew.rb` and this code block isn't needed.

2. Homebrew is set up via the test suite, in which case only the mocked `config.rb` is used and this code block isn't needed/executed either.

Is there a scenario that I am missing?